.github/workflows/unit.yml

```diff
@@ -77,16 +77,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
-      matrix:
-        node-version: [18.x, 20.x]
     steps:
       - uses: actions/checkout@v4
 
       # node setup
-      - name: Use Node.js ${{ matrix.node-version }}
+      - name: Use Node.js ${{ env.node-version }}
```

Comment:
Just checking, is the switch from matrix to env deliberate, so that only one version of node gets tested instead of two?

Reply:
deliberated, no need for the two.